pipelines/rh-advisories/README.md

```diff
@@ -23,6 +23,14 @@ the rh-push-to-registry-redhat-io pipeline.
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                              | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                                                     | No       | -                                                         |
 
+## Changes in 2.0.0
+* Add collect-atlas-params task to fetch Atlas parameters from the RPA.
+* Add create-product-sbom task to create product-level SBOMs.
+* Add update-component-sbom task to update component-level SBOMs with release
+  info.
+* Add two upload-sbom-to-atlas tasks to upload the product and updated
```

Comment:
What does this mean? In the task update, I only see one upload task being added: `upload-product-sbom`

Reply:
The README already accounts for the component changes that will be implemented when #656 is merged (as is written in the PR description above). I don't think there will be any difficulty with that, I just want to make sure no last-minute changes in #656 break this PR.

Reply:
OK, that's a very important piece information.

> Component-level SBOM upload will be added when https://github.com/konflux-ci/release-service-catalog/pull/656 is merged.

The way I read this was that once that PR is merged, the updated component sboms will be added in a subsequent PR. In that case this is really a draft - definitely not ready to be merged.

Reply:
Makes sense, I will update the description.

Reply:
I'm good with the changes here, but will let Martin resolve since it is his thread